packages/backend/src/tokens/keys.ts

```diff
@@ -116,11 +106,10 @@ export async function loadClerkJWKFromRemote({
   apiUrl = API_URL,
   apiVersion = API_VERSION,
   kid,
-  jwksCacheTtlInMs = JWKS_CACHE_TTL_MS,
```

Comment:
This doesn't seem to be functionally doing anything if beyond 5 minutes, as we have the hardcoded 5 minute max cache time.